isis/src/lro/apps/lronacpho/LROCEmpirical.cpp

```diff
@@ -333,18 +342,26 @@ namespace Isis {
    *
    * @internal
    *   @history 2016-08-15 Victor Silva - Adapted from the lrowacpho application
-                              written by Kris Becker.
+   *            written by Kris Becker.
+   *
+   *   @history 2021-03-12 Victor Silva - Added b parameters for 2019 version of
+   *            LROC Empirical algorithm
+   *
    */
   LROCEmpirical::Parameters LROCEmpirical::extract( const DbProfile &profile) const {
     Parameters pars;
-    pars.a1 = toDouble(ConfKey(profile, "A1", toString(0.0)));
-    pars.a2 = toDouble(ConfKey(profile, "A2", toString(0.0)));
-    pars.a3 = toDouble(ConfKey(profile, "A3", toString(0.0)));
+
+    for (int i=0; i<4; i++)
+        pars.aTerms.push_back(toDouble(ConfKey(profile, "A" + toString(i), toString(0.0))));
+    for (int i=0; i<7; i++)
+        pars.bTerms.push_back(toDouble(ConfKey(profile, "B" + toString(i), toString(0.0))));
+
     pars.wavelength = toDouble(ConfKey(profile, "BandBinCenter", toString(Null)));
     pars.tolerance = toDouble(ConfKey(profile, "BandBinCenterTolerance", toString(Null)));
     //  Determine equation units - defaults to Radians
     pars.units = ConfKey(profile, "Units", QString("Radians"));
     pars.phaUnit = (pars.units.toLower() == "degrees") ? 1.0 : rpd_c();
+    pars.algoVersion = toInt(ConfKey(profile, "AlgorithmVersion", QString("0")));
```

Comment:
same string question here.

Reply:
This must have gotten reverted when I corrected the merge issue. Thanks. Fixed in code.